### ISSUE TITLE
Error: Could not find API Configuration kinesisvideo-2017-09-03

### ISSUE THREAD

**Issue by tejaschandra911** [aws/aws-sdk-js#3901]
Getting the following error when I'm trying to invoke the class 'KinesisVideo'. We have upgraded the aws-sdk to the latest version i.e., version 2.996. The same version of SDK seems to work fine in Android 1.4 and react 15 but, does not work in react 17.
Please help
![Screenshot from 2021-09-28 13-56-54](https://user-images.githubusercontent.com/90621493/135205977-38a0d350-e062-4731-b5b2-41a8812c1861.png)

**Comment by ajredniwja** [comment]
HI there, unfortunately, was not able to reproduce the issue.

Code I used: 
```js
var AWS = require('aws-sdk')

var kinesisvideo = new AWS.KinesisVideo({region: 'us-west-2'});

var params = {};

kinesisvideo.listStreams(params, function(err, data) {
  if (err) console.log(err, err.stack); 
  else     console.log(data);         
});
```

Please follow the [issue template](https://github.com/aws/aws-sdk-js/issues/new?assignees=&labels=bug%2C+needs-triage&template=---bug-report.md&title=) in order for us to help you better.

**Comment by tejaschandra911** [comment]
- [x] I've gone through [Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/welcome.html) and [API reference](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/)
- [x] I've checked [AWS Forums](https://forums.aws.amazon.com) and [StackOverflow](https://stackoverflow.com/questions/tagged/aws-sdk-js) for answers
- [x] I've searched for [previous similar issues](https://github.com/aws/aws-sdk-js/issues) and didn't find any solution
- [x] This is an issue with version 2.x of the SDK

**Describe the bug**
When KinesisVideo class is instantiated, we get the error Error: Could not find API Configuration kinesisvideo-2017-09-03

**Is the issue in the browser/Node.js?**
No, not an issue with NodeJS

**If on Node.js, are you running this on AWS Lambda?**
No, its not being run on Lambda.

**SDK version number**
SDK Version: 2.996
NodeJS Version: 14.17.5
Webpack 5
React 17

**To Reproduce (observed behavior)**

**Code that we using:**
import AWS from 'aws-sdk';

const kinesisVideoClient = new AWS.KinesisVideo({
        region: formValues.region,
        accessKeyId: formValues.accessKeyId,
        secretAccessKey: formValues.secretAccessKey,
        sessionToken: formValues.sessionToken,
        endpoint: formValues.endpoint,
});

const describeSignalingChannelResponse = await kinesisVideoClient.describeSignalingChannel({
          ChannelName: formValues.channelName,
}).promise();

We getting the error when, AWS.KinesisVideo Class is being instantiated.

**Expected behavior**
Ideally, the AWS.KinesisVideo class object should allow us to execute the next set of statement but, it stops execution at AWS.KinesisVideo

**Screenshots**
Screenshot has already been shared.

**Additional context**
We are using webpack 5 with asset-modules for loading JSON. Since, the file that is being fetched by the AWS-SDK is a JSON file so, thought there could be a concern with the loader so, we moved to asset-modules for loading the JSON as, previously it was using file-loader but, that did not help.

**Comment by tejaschandra911** [comment]
@ajredniwja We managed to fix the problem. The issue was with the webpack 5 rule for JSON. Since, we were using webpack 3 and json-loader prior to the upgrade to react 17, we weren't facing this issue. We had to move to 'asset-modules' but, for some reason its not able to get the JSON file using 'asset-modules'. 

We started using 'file-loader' though, it'll be deprecated as an interim fix but, since file-loader parses the files, ../node_modules/aws-sdk/ needs to be excluded as shown in the below screen shot else, we start encountering the error shown in the initial screen shot.

We are looking for alternatives for 'file-loader' but, right now the issue seems to be fixed.
![Selection_008](https://user-images.githubusercontent.com/90621493/135504294-55034fe1-f99a-4c35-b696-2c9b0395bad3.png)
